README.md

```diff
@@ -798,10 +798,47 @@ module.exports = {
 
 For production builds it's recommended to extract the CSS from your bundle being able to use parallel loading of CSS/JS resources later on.
 
-There are two possibilities to extract a style sheet from the bundle:
+There are three possibilities to extract a style sheet from the bundle:
 
-- [mini-css-extract-plugin](https://github.com/webpack-contrib/mini-css-extract-plugin)
-- [extract-loader](https://github.com/peerigon/extract-loader) (simpler, but specialized on the css-loader's output)
+#### 1. [file-loader](https://github.com/webpack-contrib/file-loader)
```

Comment:
`file-loader` is deprecated, but you can use asset modules for this, the logic is the same

Reply:
@alexander-akait good point, I was working in Webpack 4.x when I discovered this solution. I have since updated this to utilize asset modules and it's proven compatible with Webpack 5.x. Should I just submit a new PR with that?

Reply:
We can list both of them and just mark `file-loader` solution as depcration and not reccomdend to use (and shoudl be used only for webpack v4)

Reply:
@alexander-akait I just pushed a commit which contains your recommendations, and I also stripped out irrelevant code from both examples